.travis.yml

```diff
@@ -1,14 +1,13 @@
-
-# single test suite, non-parallel build.
-
 env:
   global:
     - CC_TEST_REPORTER_ID=47b57fbbf65654b17f33b6ff4a108ce2abed31a86468033f20eb4d4e9e09935a
 language: ruby
 cache: bundler
 rvm:
   - 2.5.3
-before_install: gem install bundler -v 1.17.1
+before_install:
```

Comment:
Is there a reason we need to specify a version of bundler at all?

Reply:
TravisCI is apparently defaulting to bundler 2.0 now, and that causes the build to break (because of the `BUNDLED WITH` value in `Gemfile.lock`?).

Reply:
@jcoyne Travis recommends this line instead of ours:

` gem install bundler -v '< 2'`

Do you think that'd be preferable?

Reply:
@jcoyne Cool, that's what I'll do then.